prime-dotnet-webapi/Services/BusinessEventService.cs

```diff
@@ -71,8 +76,12 @@ public async Task<BusinessEvent> CreateAdminClaimEventAsync(int enrolleeId, stri
         }
 
 
-        private BusinessEvent CreateBusinessEvent(int BusinessEventTypeCode, int enrolleeId, string description, int? adminId = null)
+        private async Task<BusinessEvent> CreateBusinessEvent(int BusinessEventTypeCode, int enrolleeId, string description)
```

Comment:
Is every business event adding an admin id now?

Reply:
The method pulls the userId from the JWT token via the HTTP context and looks up if they are an admin. If the creator of the event is an admin, it adds the admin ID, else its null. I wanted to avoid having to pass down admin Ids everywhere, since most services don't actually need that info to do the action.

Reply:
Okay thats awesome, just wanted to make sure the same functionality was there. That is much nicer in that case!

Reply:
It will mean that we cant deliberately choose not to add the admin Id to certain events caused by admin users, but I don't think we have any need for that currently.